test/e2e-go/cli/goal/expect/statefulTealAppReadTest.exp

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/expect -f
+#exp_internal 1
+set err 0
+log_user 1
+
+source  goalExpectCommon.exp
+
+set TEST_ALGO_DIR [lindex $argv 0]
+set TEST_DATA_DIR [lindex $argv 1]
+
+proc statefulTealAppReadTest { TEST_ALGO_DIR TEST_DATA_DIR} {
+
+    set TIME_STAMP [clock seconds]
```

Comment:
This looks the same as `test/scripts/e2e_subs/e2e-app-bootloader.sh` but in `expect`, what is a difference?

Reply:
Very similar, one difference is that the wallet is specified.